Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -82,7 +91,10 @@ public struct OfflineMapAreasView: View {
                 List(models) { preplannedMapModel in
                     PreplannedListItemView(
                         model: preplannedMapModel
-                    )
+                    ) { newMap in
+                        selectedMap = newMap
+                        dismiss()
+                    }
```

Comment:
This could be simplified too:

```swift
PreplannedListItemView(model: preplannedMapModel) { newMap in
    selectedMap = newMap
    dismiss()
}
```